model/event/event_logger.go

```diff
@@ -67,7 +67,7 @@ func (l *DBEventLogger) MarkProcessed(event *EventLogEntry) error {
 	})
 	if err != nil {
 		event.ProcessedAt = time.Time{}
-		return errors.Wrap(err, "failed to update 'processed at' time")
+		return errors.Wrap(err, "updating 'processed at' time")
```

Comment:
I don't think it matters that much, it might just make the phrase flow more smoothly since it's updating "processed at" and processed is not being used as a verb.